jobs/trails/beta-3.yaml

```diff
@@ -0,0 +1,65 @@
+name: "trails-beta-3"
+version: "beta-3"
+description: "Deploy Trails contracts (beta 3)"
+
+actions:
+  - name: "trails-token-sweeper"
+    template: "erc-2470"
+    arguments:
+      salt: "{{salt-zero}}"
+      creationCode: "{{Contract(./build-info/beta-2/all-contracts.json:TrailsTokenSweeper).creationCode}}"
```

Comment:
Path looks wrong. `beta-3`?

Reply:
Fixed here: https://github.com/0xsequence/live-contracts/pull/59/commits/333851837dae3a86cf2715917178aa3e39aa28e8

Thank you Michael 🙌